regolith/helpers/l_membershelper.py

```diff
@@ -84,11 +90,37 @@ def sout(self):
             print(results, end="")
             return
 
+        for person in gtx["people"]:
+            if rc.current:
+                if not person.get('active'):
+                    continue
+                people.append(person)
```

Comment:
@berrakozer there is a duplicate `append` here!
in line 83-86, you execute `people.append(person)` if `rc.current` and  `person` in `collection` is `active`
similarly, here, you execute `people.append(person)` if `rc.current` and  `person` in `self.gtx["people"]` is `active`
since `collection = self.gtx["people"]`, you are adding the same person twice to people, hence it is printing duplicates